### PR TITLE
CT-1718 touch up open cases list

### PR DESCRIPTION
**Pull request by misaka**

### CODE REVIEW

File: spec/services/case_search_service_spec.rb [hunk 1]
```diff
@@ -153,22 +133,18 @@
...
+                         external_deadline_from_dd: external_deadline_from&.day.to_s,
+                         external_deadline_from_mm: external_deadline_from&.month.to_s,
+                         external_deadline_from_yyyy: external_deadline_from&.year.to_s,
+                         external_deadline_to_dd: external_deadline_to&.day.to_s,
+                         external_deadline_to_mm: external_deadline_to&.month.to_s,
+                         external_deadline_to_yyyy: external_deadline_to&.year.to_s,
+                       ).permit! }
         let(:search_text) { 'compliance' }
-        let(:service)     { CaseSearchService.new(user, params) }
+        # let(:service)     { CaseSearchService.new(user, params) }
```
Comment (beccapearce) [inline spec/services/case_search_service_spec.rb]: Do we still need this line?

File: spec/support/standard_setup.rb [hunk 2]
```diff
@@ -76,186 +76,186 @@ def self.method_missing(method_name, *args)
...
-      std_draft_irt:                -> { create :accepted_timeliness_review,
-                                                identifier: 'std_draft_irt' },
-      std_closed_irt:               -> { create :closed_timeliness_review,
-                                                identifier: 'std_closed_irt' }
+      std_unassigned_irt:           ->(a={}) { create(:timeliness_review,
+                                                      {identifier: 'std_unassigned_irt'}.merge(a)) },
+      std_draft_irt:                ->(a={}) { create(:accepted_timeliness_review,
+                                                      {identifier: 'std_draft_irt'}.merge(a)) },
+      std_closed_irt:               ->(a={}) { create(:closed_timeliness_review,
+                                                      {identifier: 'std_closed_irt'}.merge(a)) }
```
Comment (beccapearce) [inline spec/support/standard_setup.rb]: What's this a={}?

File: spec/models/search_query_spec.rb [hunk 3]
```diff
@@ -77,6 +77,123 @@
...
+          query_type:  'search',
+          search_text: 'courts',
+        )
+      }
+      let(:existing_search_query) {
+        create(
+          :search_query,
+          user:                   user,
+          query_type:             'search',
+          search_text:            'courts',
```
Comment (beccapearce) [inline spec/models/search_query_spec.rb]: Winnie-the-Pooh seems like a much better search term =)

File: spec/features/cases/filters/assigned_business_unit_filter_spec.rb [hunk 4]
```diff
@@ -70,8 +70,8 @@
     end
 
   end
-  
-  
+
+
```
Comment (beccapearce) [inline spec/features/cases/filters/assigned_business_unit_filter_spec.rb]: being super fussy but can we get rid of these empty lines

File: spec/services/case_search_service_spec.rb (hunk 1 above)
Comment (misaka) [reply spec/services/case_search_service_spec.rb]: Nope. Removing.

File: spec/support/standard_setup.rb (hunk 2 above)
Comment (misaka) [reply spec/support/standard_setup.rb]: This is an argument for the stabby lambda ... a is for attributes, which defaults to an empty hash.

Too obscure? The alternative is each one of these declarations turns into an even longer one, which I don't mind if this isn't obvious enough.

File: spec/models/search_query_spec.rb (hunk 3 above)
Comment (misaka) [reply spec/models/search_query_spec.rb]: I can change that. :)

Comment (misaka) [reply spec/models/search_query_spec.rb]: I can change that.

File: spec/features/cases/filters/assigned_business_unit_filter_spec.rb (hunk 4 above)
Comment (misaka) [reply spec/features/cases/filters/assigned_business_unit_filter_spec.rb]: Will do.

File: spec/support/standard_setup.rb (hunk 2 above)
Comment (beccapearce) [reply spec/support/standard_setup.rb]: I think it's fairly mystifying! you could do a comment at the top if you'd rather?

Comment (misaka) [reply spec/support/standard_setup.rb]: Sure.

Comment (misaka) [reply spec/support/standard_setup.rb]: Would turning 'a' into 'attributes' be sufficient?

Comment (beccapearce) [reply spec/support/standard_setup.rb]: yep!

Comment (misaka) [reply spec/support/standard_setup.rb]: So here's the alternative:

    @@cases = {
      std_unassigned_foi: ->(attributes={}) {
        create(:case, {identifier: 'std_unassigned_foi'}.merge(attributes))
      },
      std_awresp_foi: ->(attributes={}) {
        create(:assigned_case,
               {responding_team: responding_team,
                identifier: 'std_awresp_foi'}.merge(attributes))
      },
      std_draft_foi: ->(attributes={}) {
        create(:accepted_case,
               {responder: responder_user,
                identifier: 'std_draft_foi'}.merge(attributes))
      },
      std_draft_foi_late: ->(attributes={}) {
        create(:accepted_case,
               {responder: responder_user,
                received_date: 25.business_days.ago,
                identifier: 'std_draft_foi_late'}.merge(attributes))
      },

Comment (beccapearce) [reply spec/support/standard_setup.rb]: I know it's long but it's a fair bit clearer!

**Review by beccapearce — APPROVED** [review]

**Pull request merged**